docs/src/pages/_layout.svelte

```diff
@@ -38,7 +39,7 @@
 <!-- routify:options bundle=true -->
 <svelte:window bind:innerWidth />
 
-<Theme persist>
+<Theme persist bind:theme={$theme}>
```

Comment:
Could you run `yarn lint` here? I think it will make a style change.

Reply:
Sure.